README.md

```diff
@@ -1 +1,151 @@
-# react-capstone-metrics
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>CryptoCurrency Tracker</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 CryptoCurrency Tracker<a name="about-project"></a>
+
+> This is a webapp that can be used to track various cryptocurrency assets.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> React-Redux, React, [Coinlore API](https://www.coinlore.com/cryptocurrency-data-api)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- >
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+### Key Features <a name="key-features"></a>
+
+- **[List of Coins]**
+- **[Coin Details]**
+- **[Navigation]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://metrics-h00p.onrender.com/)
```

Comment:
- [ ] Kindly make sure to redeploy again your app because it seems not to work at the moment. The link you provided in Readme is broken. Kindly make to deploy your project again and update your Readme as well.